docs/add-ons/presentation.md

```diff
@@ -83,3 +107,55 @@ Ingress API gateway supports the following:
 - HTTP/1.x
 - Streaming content
 - Websockets
+
+Ingress grants your Add-on 2 points of security and can be implemented easily from bash as a separate, no-dependencies, background process with the default home-assistant image, as follows. Please note, this is a basic static webserver setup and provides little benefit over the log tab. Additionally, this implementation has less security than any standard HTTP Framework, but it is intended to run within a controlled environment between the shell script and the Ingress server in Home Assistant, and is not exposed directly to the Internet or any network. 
+
+Place this line somewhere near the top of your run.sh.  This will start a `nc` listening server and run the `/opt/server.sh` each time a user connects to your Ingress. The output from stdout will be displayed as a response from the server. 
+``` bash
+nc -lk -p 8099 -e  exec /opt/server.sh 3>/dev/null &
+```
+Create a `rootfs/opt` folder within your Add-on folder. Then create a `server.sh` file in that folder. The server.sh will output the HTTP Headers and HTML required to display information.
+
+/opt/server.sh
+```bash
+#HTTP Server Headers
+echo -e 'HTTP/1.1 200 OK\r\nServer: DeskPiPro\r\nDate:$(date)\r\nContent-Type: text/html; charset=UTF8\r\nCache-Control: no-store, no cache, must-revalidate\r\n\r\n'
+#HTML Document
+echo -e \
+"<!DOCTYPE html>\n"\
+"<html>\n"\
+"<head>\n"\
+"<title>Page Title</title>\n"\
+"</head>\n"\
+"<body>\n"\
+
+# HTML body content
+echo "<p>I can haz Ingress?</p>" 
+set +e   #The /tmp/status.html file may not be here if you did not create it in your addon. So we set +e to continue execution after errors.  
+cat /tmp/status.html 2>/dev/null #throw stderr to /dev/null
+set -e #Set -e to continue running in strict mode
+
+# Close up the HTML doc
+echo -e \
+"</body>\n"\
+"</html>\n"\
+"\n"
+```
+
+# Security
+
+Add-on security should be a matter of pride. You should strive for the highest level of security you can possibly attain within your skills and your Add-on limitations. If your Add-on has a lower security rating, then users will be less likely to trust it. You can use the following table to adjust your Add-on Security.
+
+| Action | Change | Notes |
+|---|---|---|
+| Use `ingress: true` in [config.json](https://developers.home-assistant.io/docs/add-ons/configuration#add-on-config) | +2 | overrides `auth_api` rating |
```

Comment:
Please, don't use full links.

Reply:
fixed in [Don't use full url.](https://github.com/home-assistant/developers.home-assistant/pull/789/commits/964527fa952faf16e52abfd59e156d2f2780d88b).